docs/guides/migration/11.0.0.md

```diff
@@ -94,7 +94,7 @@ console.log(tx.decodedEvents)
 
 ## Following contract instance methods are dropped([#1368](https://github.com/aeternity/aepp-sdk-js/pull/1368))
 * `topBlock`
-    * use `aeSdk.api.getTopHeader()` instead
+    * use `aeSdk.nodePool.api.getTopHeader()` instead
```

Comment:
```suggestion
    * use `aeSdk.api.getTopHeader()` instead
```

This is the migration guide for the older version. The new version will be v12. Can you create a new migration file `12.0.0.md` and add this information there.

Reply:
Actually, we are using `BREAKING CHANGE` commit message